frontend/__tests__/routes/_protected+/personal-information+/phone-number+/edit.test.tsx

```diff
@@ -43,6 +41,13 @@ vi.mock('~/utils/locale-utils.server', () => ({
 }));
 vi.mock('~/utils/env.server', () => ({
   featureEnabled: vi.fn().mockResolvedValue(true),
+  getEnv: vi.fn().mockReturnValue({
+    INTEROP_CCT_API_BASE_URI: 'https://api.example.com',
+    INTEROP_CCT_API_SUBSCRIPTION_KEY: '00000000000000000000000000000000',
+    INTEROP_CCT_API_COMMUNITY: 'CDCP',
+    ENGLISH_LANGUAGE_CODE: 1033,
+    FRENCH_LANGUAGE_CODE: 1036,
```

Comment:
No need to mock these particular environment variables. They aren't being used for the phone numbers feature, only letters.

Reply:
removed